pages/sign-and-certify.html

```diff
@@ -42,9 +43,9 @@
 <p>2) That any payrolls  otherwise under this contract required to be submitted for the above period are correct and complete, that the wage rates for laborers or mechanics contained therein are not less that the applicatble wage rates contained in any wage determination incorporated in to the contract, that the classifications set forth therin for each laborer or mechanic conform with the work he performed.</p>
 <p>3) That any apprentices employed in the above period are duly registered in a bona fide apprenticeship program registered with a State apprenticeship agency recognized by the Bureau of Apprenticeship and Training, United States Department of Labor, or if no such recognized agency exists in a State, are registered with the Bureau of Apprenticeship and Training, United States Department of Labor.</p>
 <p>4) That:</p>
-<p> a) WHERE FRINGE BENEFITS ARE PAID TO APPROVED PLANS, FUNDS, OR PROGRAMS</p>
+<p><input id="select-plans" type="checkbox" name="select" value="select-03" style="display: inline-block;"><label for="select-plans" style="display: inline-block;"></label> a) WHERE FRINGE BENEFITS ARE PAID TO APPROVED PLANS, FUNDS, OR PROGRAMS</p>
 <p>in addition to the basic hourly wage rates paid to each laborer or mechanic listed in the above referenced payroll, payments of fringe benefits as listed in the contract have been or will be made to appropriate programs for the benefit fo such employees, except as noted in section 4(c) below.</p>
-<p>b) WHERE FRINGE BENEFITS ARE PAID IN CASH</p>
+<p><input id="select-cash" type="checkbox" name="select" value="select-03" style="display: inline-block;"><label for="select-cash" style="display: inline-block;"></label>b) WHERE FRINGE BENEFITS ARE PAID IN CASH</p>
 <p>Each laborer or mechanic listed in the above referenced payroll has been paid, as indicated on the payroll, and amount not less than the sum of the applicable basic hourly wage rate plus the amount of the required fringe benefits as listed in the contract, except as noted in section 4(c) below.</p>
```

Comment:
@hbillings can we move the label text inside the label element so it's clickable? see: https://standards.usa.gov/form-controls/#checkboxes